cmd/uhc/whoami/cmd.go

```diff
@@ -91,23 +93,26 @@ func run(cmd *cobra.Command, argv []string) {
 		os.Exit(1)
 	}
 
-	request := connection.Get().Path("/api/accounts_mgmt/v1/current_account")
+	// Create and populate the request:
+	request := connection.AccountsMgmt().V1().CurrentAccount().Get()
+
+	// Buffer for pretty output
```

Comment:
I think this can be done later, after sending the request, otherwise it kind of interrupts the "paragraph", and makes it a bit harder to read. Note also that you don't really need the `request` variable in this case. I'd suggest something like this:

```go
// Send the request:
response, err := connection.AccountsMgmt().V1().CurrentAccount().Get().
        Send()
if err != nil {
        ...
}

// Buffer for pretty output:
buf := new(bytes.Buffer()

// Output account info:
...
```

Note also the `Send` method in a separate line. The reason I like to do that is that then it is very easy to add additional request properties. For example; imagine that you want to add a custom header to that request. You will need to just add a line:

```
// Send the request:
response, err := connection.AccountsMgmt().V1().CurrentAccount().Get().
        Header("X-My-Header", "my-value").
        Send()
if err != nil {
        ...
}
```

And when reviewing that change the _diff_ will be just one line, and very easy to review:

```diff
@@ -1,4 +1,5 @@
 request, err := connection.AccountsMgmg().V1().CurrentAccount().Get().
+	Header("X-My-Header", "my-header").
 	Send()
 if err != nil {
 	...
```